packages/react-composites/tests/browser/call/app/Utils.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { CompositeLocale, COMPOSITE_LOCALE_EN_US, COMPOSITE_LOCALE_FR_FR } from '../../../../src';
+import { QueryArgs } from './QueryArgs';
+
+export function prepareLocale(queryArgs: QueryArgs): CompositeLocale {
```

Comment:
Good question. I decided to keep it in commong code because localization tests aren't by nature hermetic (we could write the same tests as live tests).

Same with query argument parsing: Most arguments could apply to either.